www/src/components/shell/onboarding/useOnboarded.ts

```diff
@@ -9,10 +9,11 @@ import { UPDATE_USER } from '../../users/queries'
 const FORCE_ONBOARDING = 'plural-force-onboarding'
 
 function useOnboarded() {
-  const me = useContext(CurrentUserContext)
+  const { me, setRefresh } = useContext(CurrentUserContext)
 
   const [mutation] = useMutation(UPDATE_USER, {
     variables: { attributes: { onboarding: OnboardingStatus.ONBOARDED } },
+    onCompleted: () => setRefresh(true),
```

Comment:
is this not done automatically (also you can just pass the refetch function itself through the context right? instead of using the hook to do it for you)

Reply:
for context, apollo's cache should auto-update after mutations, perhaps the mutation needs to query some more fields or something

Reply:
Cache is not getting updated unfortunately. This might be a similar issue to what @maciaszczykm encountered when working on stacks loading? @maciaszczykm do you remember the exact issue there? I know it was related to cache and some fragment naming or something like that.

Reply:
> is this not done automatically (also you can just pass the refetch function itself through the context right? instead of using the hook to do it for you)

sure, I will pass it directly